modules/oc-mirror-disk-to-mirror.adoc

```diff
@@ -41,4 +41,4 @@ This command updates the mirror registry with the image set and generates the `I
 
 .Next steps
 
-* Install the `ImageContentSourcePolicy` and `CatalogSource` resources into the cluster.
+* Configure your cluster to use resources generated by oc-mirror.
```

Comment:
General comment for the PR. I now see that you use this phrase consistently throughout. Regardless of the direction you choose, please just be consistent. If you choose to use `Configure your cluster to use resources generated by oc-mirror` I think there is only 1 instance of `Configure your cluster to use resources that oc-mirror generated.` Consider updating that instance.